enterprise/backend/test/metabase_enterprise/content_translation/dictionary_test.clj

```diff
@@ -0,0 +1,173 @@
+(ns metabase-enterprise.content-translation.dictionary-test
+  "Tests for content translation dictionary import logic."
+  (:require
+   [clojure.test :refer :all]
+   [metabase-enterprise.content-translation.dictionary :as dictionary]
+   [metabase-enterprise.content-translation.utils :as ct-utils]
+   [metabase.test :as mt]
+   [metabase.util.log :as log]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+(defn- count-translations
+  "Count the number of translations in the database."
+  []
+  (count (t2/select :model/ContentTranslation)))
```

Comment:
`(t2/count :model/ContentTranslation)` is going to do a `SELECT count(*)` instead of counting the results of `SELECT *`, should be faster!